src/wallet/test/wallet_tests.cpp

```diff
@@ -778,34 +778,6 @@ BOOST_FIXTURE_TEST_CASE(CreateWallet, TestChain100Setup)
     BOOST_CHECK_EQUAL(addtx_count, 4);
 
 
-    TestUnloadWallet(std::move(wallet));
```

Comment:
471da60a5f16959480be99c77b995165b585c392  Why are you deleting this part of the test?

Reply:
It got broken. Cannot point out the reason though.

Happy to accept any help with this test.

Reply:
It was written and later refactored by @ryanofsky.